neuralop/models/padding.py

```diff
@@ -16,10 +16,11 @@ class DomainPadding(nn.Module):
     This class works for any input resolution, as long as it is in the form
     `(batch-size, channels, d1, ...., dN)`
     """
-    def __init__(self, domain_padding, padding_mode='one-sided'):
+    def __init__(self, domain_padding, padding_mode='one-sided', output_scale_factor = 1):
         super().__init__()
         self.domain_padding = domain_padding
         self.padding_mode = padding_mode.lower()
+        self.output_scale_factor = output_scale_factor
```

Comment:
I would rather have the option to use None rather than 1: we avoid any rounding issue and just skip the resampling all together if it's None